src/components/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,21 @@
+$bgColor: var(--color-gray_dark);
+
+.moonstone-tooltip_label {
+    z-index: 1;
+
+    width: inherit;
```

Comment:
Maybe we should add a `max-width` to prevent tooltip with long text to take the full width of the window
I suggest something like `max-width: 450px`